packages/components/text/src/text.tsx

```diff
@@ -203,6 +203,8 @@ Body.displayName = 'TextBody';
 export type TDetailProps = {
   isBold?: boolean;
   isItalic?: boolean;
+  // only span is supported for now.
+  as?: 'span';
```

Comment:
I realise now that we switch the default render to `div` and had `display: block` removed.
Given that `<small>` is an inline element, we can also accept it here.

Reply:
Yep